### DISCUSSION TITLE
Surface Laptop 3 Intel Features

### DISCUSSION THREAD

**Discussion by Austin4705** [linux-surface/linux-surface#874] [General]
Out of curiosity before switching, are there any plans/how close is the surface laptop 3 intel version towards getting sensors and pen to work properly. Also, is the sensor the ambient light sensor, or is there any other sensor the thing is talking about? Also, I was wondering about boot times or anything noteworthy in SL 3intel experience on Linux if anyone used it.

**Comment by qzed** [comment]
I think the work  at https://github.com/linux-surface/intel-precise-touch/issues/14 should also apply to the pen. Sensors generally include ambient light sensor (which should work, see https://github.com/linux-surface/linux-surface/issues/121) and other sensors like gyro/accel for device orientation. I don't know whether the latter work (or if they're even present on the device). According to the feature matrix they don't work.
